docs/resources/public_ip_associate.md

```diff
@@ -0,0 +1,40 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "stackit_public_ip_associate Resource - stackit"
+subcategory: ""
+description: |-
+  Associates an existing public IP to a network interface. This is useful for situations where you have a pre-allocated public IP or unable to use the stackit_public_ip resource to create a new public IP. Must have a region specified in the provider configuration.
+  The stackit_public_ip_associate resource should never be used together with the stackit_public_ip resource.Both resources have control of the stackit_network_interface association. If used together, this will lead to conflicts.
```

Comment:
There a white space missing after "[...] stackit_public_ip resource.Both [...]"

Reply:
Hi @h3adex, thanks for the review and feedback. I added the missing space between sentences.